packages/card/src/__demo__/index.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import cardExamples from './examples/cardIndex';
+import cardActionsExamples from './examples/cardActionsIndex';
+import cardBlockExamples from './examples/cardBlockIndex';
+import cardImageExamples from './examples/cardImageIndex';
+import cardTitleExamples from './examples/cardTitleIndex';
+
+const cardDoc = require('!!react-docgen-loader!../Card.js');
```

Comment:
Aww, bummer. I really liked the elegance of directly exporting the output of `createStyledComponent`, but you're right, it totally breaks docgen-loader.